int64 num_of_fingers
---
grasp_execution/graspArr grasps
